features/step_definitions/extra_steps.rb

```diff
@@ -153,6 +153,14 @@
   expect(petition).to exist
 end
 
+Then(/^a petition should exist with attributes (.*)$/) do |str|
+  attributes = eval("{#{str}}")
```

Comment:
There's a better way to do this - create a specific step for those set of attributes like how we did when removing pickle